Gemfile

```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
-gem "github-pages", group: :jekyll_plugins
\ No newline at end of file
+gem "github-pages", group: :jekyll_plugins
+gem 'jekyll-sass-converter'
```

Comment:
@captainyakult unfortunately Github Pages doesn't support any third party plugins outside of the official ones:
https://help.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll#default-plugins

> Note that GitHub Pages runs in safe mode and only allows a set of whitelisted plugins.

From: https://jekyllrb.com/docs/github-pages/


We have 2 options:

1.  Generate the site locally and then push the site's static files to GitHub. Kind of annoying cause we would always have to remember to do it.
2. Don't use unsupported plugins.

Do we even need that plugin, isn't SASS baked into Jekyll already: https://jekyllrb.com/docs/step-by-step/07-assets/#sass

Reply:
> Jekyll Sass Converter requires Jekyll 2.0.0 or greater and is bundled with Jekyll so you don't need to install it if you're already using Jekyll.

From https://github.com/jekyll/jekyll-sass-converter

Reply:
Remove jekyll-sass-converter from config and site is still building and working.